_data/navbar.yml

```
@@ -10,8 +10,10 @@
 - name: How it works
   index: 3
   subfolderitems:
-      - page: Timeline
+      - page: LONG-TERM FORMAT TIMELINE
```

In order to avoid the breaking line, I suggest making a bit short the title, for example:

```suggestion
      - page: LONG-TERM TIMELINE
```